pgrustxn/src/texttools.rs

```diff
@@ -0,0 +1,55 @@
+#[cfg(pg94)]
+pub use pg94 as pg;
+
+//#[cfg(pg95)]
+pub use pg95 as pg;
+
+use std::ffi::CString;
+use std::mem::size_of;
+
+pub fn write_text1(ctx: *mut pg::Struct_LogicalDecodingContext,
```

Comment:
This is a good direction to go. Is `t` a text object? Like a `StringInfo` or something like that?